### PR TITLE
Add my first change to commands.ts.  This is a great commit message.

### PR DESCRIPTION
**Pull request by spbrogan**
Here is more info for the commit.  Hopefully it shows up nice.

### CODE REVIEW

**Comment by spbrogan** [comment]
Lets add a comment here and make sure matt handled this case.  good luck

File: extension/commands.ts [hunk 1]
```diff
@@ -33,6 +33,8 @@ export class UefiCommands implements vscode.Disposable {
     dispose() {
     }
 
+    Lets add a change here.
```
Comment (spbrogan) [inline extension/commands.ts:36]: I don't like this change.

File: extension/logger.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-import { TransformableInfo } from 'logform';
+Adding a second commit.
```
Comment (spbrogan) [inline extension/logger.ts:3]: this change doesn't look valid either

**Review by spbrogan — COMMENTED** [review]
Overall this change has a lot of whitespace changes and invalid syntax.  I don't think it should be merged.   Just my comment.  :)

File: extension/commands.ts (hunk 1 above)
Comment (spbrogan) [reply extension/commands.ts:36]: This didn't show up.  @matthewfcarlson  needs to review how he handles PR review comments.

File: extension/commands.ts [hunk 3]
```diff
@@ -33,6 +33,8 @@ export class UefiCommands implements vscode.Disposable {
     dispose() {
```
Comment (spbrogan) [inline extension/commands.ts]:
```suggestion
    dispose() { wow..can i really submit code changes here 
```

File: extension/commands.ts (hunk 1 above)
Comment (matthewfcarlson) [reply extension/commands.ts:36]: I'm working on it :)

File: extension/logger.ts (hunk 2 above)
Comment (matthewfcarlson) [reply extension/logger.ts:3]: now it should work?

File: extension/commands.ts (hunk 1 above)
Comment (matthewfcarlson) [reply extension/commands.ts:36]: Here we go :)

Comment (matthewfcarlson) [reply extension/commands.ts:36]: There we go :)

Comment (matthewfcarlson) [reply extension/commands.ts:36]: This is working

File: extension/commands.ts [hunk 4]
```diff
@@ -166,9 +168,9 @@ export class UefiCommands implements vscode.Disposable {
         catch {
             logger.info("Failed to install requirements");
             await this.term.runPythonCommand(["-m", "pip", "install", "--user", "--upgrade", "mu-build"]);
-            //installing mu_build will install all the needed dependencies 
+            //installing mu_build will install all the needed dependencies
```
Comment (matthewfcarlson) [inline extension/commands.ts:171]: Now we're cooking with gas

Comment (matthewfcarlson) [reply extension/commands.ts:171]: Haha

Comment (matthewfcarlson) [reply extension/commands.ts:171]: Wheee

File: extension/logger.ts (hunk 2 above)
Comment (matthewfcarlson) [reply extension/logger.ts:3]: hey

File: extension/commands.ts (hunk 4 above)
Comment (matthewfcarlson) [reply extension/commands.ts:171]: ?

**Pull request closed without merging**